specification/azsadmin/resource-manager/user-subscriptions/Microsoft.Subscriptions/preview/2015-11-01/Subscriptions.json

```diff
@@ -0,0 +1,412 @@
+        "description": "The User Subscriptions Management Client."
+    },
+    "host": "management.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the list of subscriptions.": {
+                        "$ref": "./examples/Subscriptions/List.json"
+                    }
+                },
+                "description": "Get the list of subscriptions.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "Subscriptions_List",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/SubscriptionList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the specified subscription.": {
+                        "$ref": "./examples/Subscriptions/Get.json"
+                    }
+              },
+              "tags": [
+                "Subscriptions"
+              ],
+              "operationId": "Subscriptions_Get",
+              "description": "Gets details about particular subscription.",
+              "parameters": [
+                {
+                  "name": "subscriptionId",
+                  "in": "path",
+                  "required": true,
+                  "type": "string",
+                  "description": "Id of the subscription."
+                },
+                {
+                  "$ref": "#/parameters/ApiVersionParameter"
+                }
+              ],
+              "responses": {
+                "200": {
+                  "description": "",
+                  "schema": {
+                    "$ref": "#/definitions/Subscription"
+                  }
+                }
+              }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Create or updates a subscription": {
+                        "$ref": "./examples/Subscriptions/Create.json"
+                    }
+                },
+                "description": "Create or updates a subscription.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "Subscriptions_CreateOrUpdate",
+                "parameters": [
+                    {
```

Comment:
Why are you not using the SubscriptionIdParameter here?

Reply:
Subscription is the main resource for this spec, I dont want to mask it in global param